ppx/ppx_js/lib_internal/ppx_js_internal.ml

```diff
@@ -241,7 +250,7 @@ let invoker ?(extra_types = []) uplift downlift body arguments =
   let labels_and_pats =
     List.map arguments ~f:(fun d ->
         let label = Arg.label d in
-        let patt = Pat.var (Location.mknoloc (Arg.name d)) in
+        let patt = Pat.var { txt = Arg.name d; loc = Location.none } in
```

Comment:
can we introduce mknoloc and mkloc to minimize the diff ?

Reply:
Yes, I've done that now.